cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModelExt.kt

```diff
@@ -131,26 +131,25 @@ internal fun MessagesParamReq.toConsentStatusParamReq(
     )
 }
 
-internal fun CCPA.toCcpaCS() = CcpaCS(
+internal fun CCPA.toCcpaCS(applies: Boolean?) = CcpaCS(
```

Comment:
now I'm thinking, wouldn't it be better for us not to update `applies` value anywhere, except from the response of the `/meta-data` call? that will ease-out this flow, so that we don't need to pass an extra value into the ext function, and we will not worry about the value of `applies` having wrong value

wdyt?